policyengine_us/tests/policy/baseline/gov/states/va/tax/income/va_income_tax_before_non_refundable_credits.yaml

```
@@ -39,6 +39,14 @@
   period: 2022
   input:
     va_taxable_income: 30_000
-    state_code: VA
+    va_must_file: true
   output:
     va_income_tax_before_non_refundable_credits: 1_467.5
+
+- name: VA must not file
```

Since I think they still could
```suggestion
- name: VA does not need to file
```